matchzoo/preprocessors.py

```diff
@@ -228,3 +228,70 @@ def fit(self, tokens: list, ngram: int=3):
         """
         _, input_dim = self._create_n_letters(tokens, ngram)
         self._state = {'input_dim': input_dim}
+
+
+class Vocabulary(StatefulProcessorUnit):
```

Comment:
can we use the same naming convention indicates that it's a `ProcessorUnit`? e.g. `VocabularyUnit`

Reply:
Sure.